doc/object_properties.adoc

```diff
@@ -0,0 +1,252 @@
+= Object Properties
+include::../include/config.adoc[]
+:url-c-content-src: {url-engine-src}/script/common/c_content.cpp
+
+WARNING: There may be missing/incorrect information here,
+please refer to the {url-lua-api}#L7141-L7304[Lua API]
+and the {url-c-content-src}#L186-L442[relevant engine code]
+
+WARNING: This is missing the default values for everything
```

Comment:
Add them please

Reply:
where would i find them?

Reply:
object_properties.h is a good place to start, but be careful - the internal defaults there may be overridden somewhere else.